src/test/scala/io/iohk/ethereum/vm/MockWorldState.scala

```diff
@@ -1,7 +1,7 @@
 package io.iohk.ethereum.vm
 
 import akka.util.ByteString
-import io.iohk.ethereum.domain.{Account, Address}
+import io.iohk.ethereum.domain.{Account, Address, BlockHeader}
```

Comment:
it is used by headers: `Map[BigInt, BlockHeader] = Map()`, but this case go as well